packages/design/tailwind/css/components.css

```diff
@@ -101,25 +101,26 @@
 /* Tag */
 
 .gi-tag {
-  @apply gi-rounded-md gi-border-xs gi-font-normal gi-py-0 gi-px-2 gi-inline-flex gi-justify-center gi-items-center;
+  @apply gi-rounded-sm gi-border-xs gi-font-bold gi-leading-6
```

Comment:
can we fix the spacing here ? it seems to be on 2 lines

Reply:
Good catch!